python/test/unit/language/test_core.py

```diff
@@ -220,6 +220,8 @@ def is_layout_applicable(layout) -> bool:
     common_layouts = [BlockedLayout, SharedLayout]
     if layout in common_layouts:
         return True
+    elif isinstance(layout, BlockedLayout) or isinstance(layout, SharedLayout):
```

Comment:
```suggestion
    if isinstance(layout, (BlockedLayout, SharedLayout)):
```

This could be better

Reply:
Changed, thanks for the suggestion.